assets/src/edit-story/components/panels/index.js

```diff
@@ -34,27 +35,29 @@ export { default as ColorPresetPanel } from './colorPreset';
 const BACKGROUND_SIZE_POSITION = 'backgroundSizePosition';
 const BACKGROUND_DISPLAY = 'backgroundDisplay';
 const BACKGROUND_OVERLAY = 'backgroundOverlay';
+const IMAGE_ACCESSIBILITY = 'imageAccessibility';
 const LAYER_STYLE = 'layerStyle';
 const LINK = 'link';
 const TEXT = 'text';
 const SIZE_POSITION = 'sizePosition';
 const STYLE = 'style';
 const TEXT_STYLE = 'textStyle';
-const VIDEO_POSTER = 'videoPoster';
+const VIDEO_ACCESSIBILITY = 'videoAccessibility';
 const PAGE = 'page';
 const NO_SELECTION = 'noselection';
 
 export const PanelTypes = {
   BACKGROUND_SIZE_POSITION,
   BACKGROUND_DISPLAY,
   BACKGROUND_OVERLAY,
+  IMAGE_ACCESSIBILITY,
```

Comment:
This should probably move to the bottom, as per the order in Figma

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/652/commits/eef36fbe3357f8343b08bf9f4daa513550c45378